src/components/Modal/Modal.tsx

```diff
@@ -127,8 +119,8 @@ const Modal: FC<ModalProps> = ({
             <Input
               type="datetime-local"
               value={modalData.createdDate}
-              min={getTenMinAgo()}
-              max={getTenYearsAfter()}
+              min={getCorrectDateStr(-10)}
+              max={getCorrectDateStr(10 * 365 * 24 * 60)}
```

Comment:
As I told - magic numbers

Reply:
Fixed - create enum for time delay and pass them as arguments in function